addons/web_editor/static/src/js/wysiwyg/widgets/link_tools.js

```diff
@@ -440,6 +440,7 @@ export class LinkTools extends Link {
         this._setSelectOption($target, true);
         this._updateOptionsUI();
         this._adaptPreview();
+        this.linkEl.click();
```

Comment:
```suggestion
        this.props.wysiwyg.snippetsMenu.activateSnippet($(this.linkEl));
```
We should avoid triggering events on elements for things like that and instead use the appropriate function 😉
Can you also add a comment to explain why this line is here (to make sure it doesn't get removed) 🙏🏼

Reply:
Thank for the review, i actually didn't know how to trigger that. I applied the changes.